components/availability/src/Availability.svelte

```diff
@@ -501,9 +501,8 @@
       return slotList;
     }, [] as SelectableSlot[]);
 
-  $: if (sortedSlots.length) {
-    dispatchEvent("timeSlotChosen", { timeSlots: sortedSlots });
-  }
+  $: dispatchEvent("timeSlotChosen", { timeSlots: sortedSlots });
```

Comment:
Side-effect: removed the .length condition here so we can reset slots down to 0 and show that in the scheduler (in prod, try removing the # of events down to 0; it'll still show the last-selected one)